velox/exec/tests/utils/AssertQueryBuilder.h

```diff
@@ -175,6 +175,9 @@ class AssertQueryBuilder {
       memory::MemoryPool* pool,
       std::shared_ptr<Task>& task);
 
+  /// Only run the query, no check for the result.
+  std::shared_ptr<Task> runQuery();
```

Comment:
'Query' is not necessary since it is a method in XxxQueryXxx class.

I wonder if 'run' could be confusing as it is not clear whether the method just starts running and expects the caller to monitor / wait for completion or the method actually waits for completion.

Perhaps, runToCompletion() or countResults() is clearer.

```
std::shared_ptr<Task> runToCompletion();

int64_t countResults(std::shared_ptr<Task>& task);
```